README.md

```diff
@@ -110,17 +110,57 @@ to this:
 DATABASES = {
     "default": {
         "ENGINE": "django_mongodb",
+        "HOST": "mongodb+srv://cluster0.example.mongodb.net",
```

Comment:
Is it correct to say that the scheme is required in HOST only for srv connections? (i.e. `mongodb://` is never required -- as tests seem to indicate?)

Reply:
Yes, and in `parse_uri` I think we only add scheme to host if srv connection.

Reply:
I will add: 
```
`HOST` only needs a scheme prefix for SRV connections (`mongodb+srv://`). A
`mongodb://` prefix is never required.
```